azure-pipelines.yml

```diff
@@ -11,40 +11,52 @@ jobs:
   - script: |
       pip install -U docker-compose conan conan-package-tools
       cd modern && python run.py
+      pip uninstall --yes conan conan-package-tools
+      ./modern/run_pytest.sh
     env:
       DOCKER_LOGIN_USERNAME: $(DOCKER_LOGIN_USERNAME)
       DOCKER_PASSWORD: $(DOCKER_PASSWORD)
   strategy:
     matrix:
       Ubuntu 16.04 GCC 5 x86_64:
         GCC_VERSIONS: "5"
+        GCC_VERSION: "5"
```

Comment:
The CI won't build several images in the same matrix entry. Users should prefer known docker commands over magic hidden in python scripts. If they want to build several images, they can just concatenate several commands. Avoid complexity on our side.

Reply:
Okay, let keep this PR more focused on tests. I can do it directly in single-image branch. It also required README update.